rpc/pb/transaction.proto

```diff
@@ -39,23 +46,23 @@ service TransactionCommand {
         };
     }
 
-    rpc GetTokenBalance(GetTokenBalanceRequest) returns (GetTokenBalanceResponse) {
+    rpc GetFeePrice(GetFeePriceRequest) returns (GetFeePriceResponse) {
         option (google.api.http) = {
-            post: "/v1/tx/gettokenbalance"
+            post: "/v1/tx/getfeeprice"
             body: "*"
         };
     }
 
-    rpc GetFeePrice(GetFeePriceRequest) returns (GetFeePriceResponse) {
+    rpc GetTransactionPool(GetTransactionPoolRequest) returns (GetTransactionsResponse) {
         option (google.api.http) = {
-            post: "/v1/tx/getfeeprice"
+            post: "/v1/tx/gettxpool"
```

Comment:
What does the `gettxpool`  mean?  pls give  a more meaningful method name. 
`get_pendding_txs`?